### DISCUSSION TITLE
Informal takedown request

### DISCUSSION THREAD

**Discussion by luphoria** [3kh0/ext-remover#165] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello.
This repository is sharing public information from TitaniumNetwork on GitHub.
There is nothing truly wrong with this; however, it _is_ promoting real-world use of these tools. In an actual school environment, utilizing an unenrollment tool is completely and entirely illegal - theft - and I'm not being cheeky or joking when I say that the tools exist as proof-of-concept and for educational purposes. This repository promotes the active use of these tools to hack devices that are owned by the school, and I do not think that is right.
The write-up you've created is appreciated, but doesn't properly explain the "LTBEEF" exploit in any regard - nor is there a write-up for the unenrolling tool. 
Furthermore, you have attached a license to these tools - the WTFPL. You are not the rights-holder to these tools, which do not have the WTFPL attached to them.
All in all, I think this repository exists in bad faith, and I am requesting that you take it down on your own. I will not pursue legal action - and I don't think anything illegal happened except the licensefile - but please understand the activity you are promoting with this repository.

**Comment by synapt0** [comment]
Watashi no na wa Kira Yoshikage. Renrei sanjyuu sansai. I am here to inform you that we do not care.

**Comment by 3kh0** [comment, marked as answer]
@luphoria Hit me up on discord.

**Reply by Aka-but-nice** [reply]
From Now on, No more telling, people to unenroll.

**Reply by rwg100** [reply]
hitting people is not a good idea, maybe you should talk it out

**Reply by nathblade16** [reply]
yes, we probably shouldn't hit people

**Comment by bartholemuew** [comment]
bla bla bla....

**Comment by ghost** [comment]
Well, ltbeef is fine but not the unenrolling tool which violates policies

**Comment by TheMemeSniper** [comment]
Guys! Didn't y'all believe that all those puzzle ads were fake, and that there's no such game? But today, I've FINALLY FOUND THE GAME. Evony: The King's Return. That's right guys, that's the exact game that you always see in those ads! I've seen enough of their videos, today I'm gonna play it. To be honest, I'm gonna use my brains to play this. OHH, I failed again! Look! Now I'm stuck on this level! Oh my god I lost my treasure! Time to use your brain. Download it now and give it a try.

**Reply by Aka-but-nice** [reply]
> Guys! Didn't y'all believe that all those puzzle ads were fake, and that there's no such game? But today, I've FINALLY FOUND THE GAME. Evony: The King's Return. That's right guys, that's the exact game that you always see in those ads! I've seen enough of their videos, today I'm gonna play it. To be honest, I'm gonna use my brains to play this. OHH, I failed again! Look! Now I'm stuck on this level! Oh my god I lost my treasure! Time to use your brain. Download it now and give it a try.

Reminds, me of a particular ad💀

**Comment by deryilz** [comment]
Man, you know what's the worst thing about being a student these days? Securly. That's right, Securly. It's like the government is constantly watching you. Like, I know I'm not doing anything wrong, but it still feels like Big Brother is always watching. I can't even go on Instagram during class without Securly flagging it. It's like they don't trust us to do anything. It's like they think we're all just a bunch of delinquents. I'm telling you, it's like a prison.

**Reply by Aka-but-nice** [reply]
@DerDer56  aren't you Bypassi

**Reply by TheMemeSniper** [reply]
> @DerDer56  aren't you Bypassi 

All Chromebooks post-2015 contain an enrollment chip. This enrollment chip is a large chip under a thermal pad; once you remove it, the Chromebook will be unenrolled. This should not be a difficult process, just pull it off with a high level of force (do not worry, you will not damage anything). However, if you do end up damaging something, even with a screwdriver, it will be enough to wipe enrollment, but don't file a lawsuit against me for any liabilities you may incur.

**Reply by Aka-but-nice** [reply]
Why you reply to me?

**Reply by Alex2Mayhem** [reply]
> > @DerDer56  aren't you Bypassi
> 
> All Chromebooks post-2015 contain an enrollment chip. This enrollment chip is a large chip under a thermal pad; once you remove it, the Chromebook will be unenrolled. This should not be a difficult process, just pull it off with a high level of force (do not worry, you will not damage anything). However, if you do end up damaging something, even with a screwdriver, it will be enough to wipe enrollment, but don't file a lawsuit against me for any liabilities you may incur.


Im gonna pull your unenrollment chip. Clearly stated to not unenroll.

**Reply by RuralAnemone** [reply]
Man, you know what's the worst thing about being a student these days? Securly. That's right, Securly. It's like the government is constantly watching you. Like, I know I'm not doing anything wrong, but it still feels like Big Brother is always watching. I can't even go on Instagram during class without Securly flagging it. It's like they don't trust us to do anything. It's like they think we're all just a bunch of delinquents. I'm telling you, it's like a prison.

**Reply by rwg100** [reply]
why would you be on insta in class? you stating that they dont trust you to do anything during class, but then you want to go on instagram? no wonder they dont let you have any freedom.

**Reply by RuralAnemone** [reply]
> why would you be on insta in class? you stating that they dont trust you to do anything during class, but then you want to go on instagram? no wonder they dont let you have any freedom.

Man, you know what's the worst thing about being a student these days? Securly. That's right, Securly. It's like the government is constantly watching you. Like, I know I'm not doing anything wrong, but it still feels like Big Brother is always watching. I can't even go on Instagram during class without Securly flagging it. It's like they don't trust us to do anything. It's like they think we're all just a bunch of delinquents. I'm telling you, it's like a prison.
